esptfa_arima/Test_Management/views.py

```
@@ -418,6 +448,56 @@ def get_queryset(self):
         return StudentScoresStatistic.objects.filter(analysis_document_id=analysis_document_id)
 
 
+class ActualPostTestViewSet(viewsets.ModelViewSet):
+    queryset = ActualPostTest.objects.all()
```

Ruff (0.14.10)

`ActualPostTest` may be undefined, or defined from star imports

(F405)